_posts/2016-05-16-exploring-rx-operators-map.md

```diff
@@ -86,11 +91,14 @@ let obs = Rx.Observable.fromEvent(demoInput, 'input')
 {% endraw %}
 {% endhighlight %}
 
-Of course we could have done the same in the first `map` call already but it's sometimes more readable to break things into multiple steps. We also often use different operators in between of two `map` calls (e.g to filter something out).
+Of course, we could have done the same in the first `map` call. But it's sometimes more readable to break things into multiple steps. Notice that often we also use different operators in between of two `map` calls (e.g to filter something out).
 
 If you like to play a bit with the operator yourself, here is a working demo.
 
 <iframe src="http://embed.plnkr.co/iWR9b2s4wd0pZSOEfHuR/"></iframe>
 
+At this point, you may think that Observables are really just a minor enhancement on the Observer or Futures patterns... better suited to handle a sequence of events rather than a single callback. And the `.map()` function certainly does not - at first glance - seem to offer any added-value. The power of Observables is revealed when you start using Rx operators to transform, combine, manipulate, and work with sequences of items emitted by Observables. 
```

Comment:
Correct, "Futures" might confuse ppl that are not coming from Java land or other languages where this terminology is a thing